queries.sql

```diff
@@ -51,4 +51,29 @@ SELECT owners.full_name, animals.name FROM owners LEFT JOIN animals ON owners.id
 SELECT species.name, COUNT(*) FROM animals JOIN species ON animals.species_id = species.id GROUP BY species.name;
 SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id JOIN species ON animals.species_id = species.id WHERE owners.full_name = 'Jeniffer Orwell' AND species.name = 'Digimon';
 SELECT animals.name FROM animals JOIN owners ON animals.owner_id = owners.id WHERE owners.full_name = 'Dean Winchester' AND animals.escape_attempts = 0;
-SELECT owners.full_name, COUNT(animals.id) AS num_owned FROM owners LEFT JOIN animals ON owners.id = animals.owner_id GROUP BY owners.id, owners.full_name ORDER BY num_owned DESC;
\ No newline at end of file
+SELECT owners.full_name, COUNT(animals.id) AS num_owned FROM owners LEFT JOIN animals ON owners.id = animals.owner_id GROUP BY owners.id, owners.full_name ORDER BY num_owned DESC;
+
+SELECT animals.name FROM animals JOIN visits ON animals.id = visits.animal_id JOIN vets ON visits.vet_id = vets.id WHERE vets.name = 'William Tatcher' ORDER BY visits.visit_date DESC;
+SELECT COUNT(DISTINCT visits.animal_id) AS num_animals_seen FROM visits JOIN vets ON visits.vet_id = vets.id WHERE vets.name = 'Stephanie Mendez';
+SELECT vets.name, species.name FROM vets LEFT JOIN specializations ON vets.id = specializations.vet_id LEFT JOIN species ON specializations.species_id = species.id ORDER BY vets.id, species.id;
```

Comment:
- [ ] Kindly list the correct values for the vets and their specialties.

List all vets and their specialties, including vets with no specialties.
Answer:

William Tatcher (Specialty: Pokemon)
Maisy Smith (No Specialties)
Stephanie Mendez (Specialties: Pokemon, Digimon)
Jack Harkness (Specialty: Digimon)